src/test/java/seedu/address/model/session/RecurringSessionTest.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.model.session;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class RecurringSessionTest {
+
+    @Test
+    void getInterval() {
+    }
+
+    @Test
+    void getEndDateTime() {
+    }
```

Comment:
Hi, maybe we could add a //TODO: tag here, so that we would remember to come back here in the near future and work on this.

Reply:
Thanks, did in the coming commit